app/controllers/pages_controller.rb

```diff
@@ -46,11 +46,11 @@ def update
   end
 
   def destroy
-    @page = Page.find(params[:id])
+    @page = Page.friendly.find(params[:id])
 
     if @page.destroy
       flash[:notice] = "\"#{@page.title}\" was deleted successfully."
-      redirect_to pages_path
+      redirect_to root_path
```

Comment:
Same stuff here: fixing the redirection root has nothing to do with fixing the `friendly` finders, so making a habit of using `git staged` can help you detect that.

In that specific case, because both concerns affect the same file, we could have used [`git add -p` to selectively stage chunks](http://johnkary.net/blog/git-add-p-the-most-powerful-git-feature-youre-not-using-yet/) from that file, creating two separate commits for improved readability.